### PR TITLE
Add levenshtein@1.0.5 w/ npm auto-update

### PR DESCRIPTION
**Pull request by x09326**
@pvnr0082t @kennynaoh please help me review it.
Manually add v1.0.3 from git repo because npm auto-update can't get it.
And I didn't add the homepage because I can't access it.

Pull request for issue: #10169 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/gf3/Levenshtein
 * Official website (optional, not the repository):https://gf3.github.io/Levenshtein/
 * NPM package url (optional):https://www.npmjs.com/package/levenshtein
 * License and its reference:Unlicense, https://github.com/gf3/Levenshtein/blob/master/UNLICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of stars:224
   - Count of watchers:12
   - Count of forks:34
 * NPM download stats (optional):
   - Downloads in the last day:1034
   - Downloads in the last week:6101
   - Downloads in the last month:21715

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/levenshtein/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "levenshtein",
+  "filename": "levenshtein.min.js",
+  "description": "Javascript implementation of the L-diggity.",
+  "author": "Gianni Chiappetta <gianni@runlevel6.org> (http://gf3.ca)",
```
Comment (kennynaoh) [inline ajax/libs/levenshtein/package.json]: `http://gf3.ca` this site can't be reached.

Comment (kennynaoh) [inline ajax/libs/levenshtein/package.json]: `http://gf3.ca` this site can't be reached.

**Comment by x09326** [comment]
@pvnr0082t I have explained it in the third line

**Comment by x09326** [comment]
@kennynaoh I have removed the url in the `author` field.

File: ajax/libs/levenshtein/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "levenshtein",
+  "filename": "levenshtein.min.js",
+  "description": "Javascript implementation of the L-diggity.",
+  "author": "Gianni Chiappetta <gianni@runlevel6.org> ",
```
Comment (kennynaoh) [inline ajax/libs/levenshtein/package.json]: @x09326 Please remove the blank in the end.

**Comment by x09326** [comment]
@kennynaoh I have removed it.

**Pull request merged**